modules/placement-driver/src/main/java/org/apache/ignite/internal/placementdriver/LeaseUpdater.java

```diff
@@ -181,22 +210,20 @@ public void run() {
 
                     HybridTimestamp now = clock.now();
 
-                    // Nothing holds the lease.
-                    if (lease == EMPTY_LEASE
-                            // The lease is near to expiration.
-                            || now.getPhysical() > (lease.getLeaseExpirationTime().getPhysical() - LEASE_PERIOD / 2)) {
+                    // The lease is expired or near to the one.
+                    if (now.getPhysical() > (lease.getExpirationTime().getPhysical() - LEASE_PERIOD / 2)) {
```

Comment:
I don't sure that clock skew is required here. Because here we are doing a check proactive comparison, far a way from so little interval like a clock skew is.